### PR TITLE
Fixes a bug with queued commands not working

### PR DESCRIPTION
**Pull request by jakofranko**
Hi there, I was playing around with Ronin and noticed that chaining commands via `;` was not working most of the time (but sometimes). 

tl;dr - duplicate event listeners were clearing the queued commands prematurely.

When stepping through the code, I saw that when I executed the command chain via the "Enter" key, it would hit this switch statement:

```
switch (event.key) {
      case "Enter": this.key_enter(); break;
      ...
}
```

and queue up the commands successfully. Then, as that event-listener function would continue, it would hit this:

```
switch(event.which) {
      case 13:  this.key_enter();  break;
      ...
}
```

and try to execute the queue again, but since the user input at this point would be empty, it would clear out the queue and keep the chained commands from being executed.

I noticed there were a couple of other duplicated keys in this second block which could cause problems too, so I removed them as well...

Let me know if I'm totally missing something, but this seems to fix the problem.

EDIT: further clarification -- `key_enter()` runs `commander.query()` which builds the commands into the globally declared `queue` variable. `commander.run()` iterates through the queue on a timer, and it was in-between executions of these timed executions of `run()` (line 32 of commander.js) that the queue was being wiped out by the duplicated `key_enter()` function.

### CODE REVIEW

**Comment by neauoire** [comment]
It should be working properly now, sorry.
The queue is now handled with the Terminal module.
